text/profiles/0212-profiling-vision.md

```diff
@@ -0,0 +1,96 @@
+- **Profiling should be vendor neutral**: From one profiling agent, users should be able to send data to whichever vendor they like (or no vendor at all) and interoperate with other OSS projects
+
+## Current State of Profilers
+
+As it currently stands, the method for collecting profiles for an application and the format of the profiles collected varies greatly depending on several factors such as:
+
+- Language (and language runtime)
+- Profiler Type
+- Data type being profiled (i.e. cpu, memory, etc)
+- Availability or utilization of symbolic information
+
+A fairly comprehensive taxonomy of various profiling formats can be found on the [profilerpedia website](https://profilerpedia.markhansen.co.nz/formats/).
+
+As a result of this variation, the tooling and collection of profiling data lacks in exactly the areas in which OpenTelemetry has built as its core engineering values:
+
+- Profiling currently lacks compatibility: Each vendor, open source project, and language has different ways of collecting, sending, and storing profiling data and often with no regard to linking to other signals
+- Profiling currently lacks consistency: Currently profiling agents and formats can change arbitrarily with no unified criteria for how to take end-users into account
+
+## Making Profiling Compatible with other Signals
+
+Profiles are particularly useful in the context of other signals. For example, having a profile for a particular “slow” span in a trace yields more actionable information than simply knowing that the span was slow. The nature of profiling also provides the potential for a more “hands-off” approach to adding it to code compared to manual instrumentation needed for other signals.
+
+OpenTelemetry will define how profiles will be correlated with logs, traces, and metrics and how this correlation information will be stored.
+
+Correlation will work across 2 major dimensions:
+
+- To correlate telemetry emitted for the same request (also known as request or trace context correlation)
+- To correlate telemetry emitted from the same source (also known as Resource Context Correlation)
+
+## Standardize profiling data model for industry-wide sharing and reuse
+
+We will design a profiling data model that will aim to represent the vast majority of profiling data with the following goals in mind:
+
+- Profile format should be as compact as possible
+- Profiling data should be transferred as efficiently as possible and the model should be lossless with intentional bias for enabling efficient marshaling, transcoding, and analysis
+- When needed, existing profiling formats should be able to be unambiguously mapped to the standardized data model (i.e. collapsed, pprof, JFR, etc.)
+- Providing minimal/terse data model components that show relationships between other telemetry components. For example, linking call stacks with spans
```

Comment:
Editorial: this one is a bit difficult to understand. It is unlike the previous 3 bullet points, which list what "should be". Can you make this clearer ([this doc about parallel vs nonparallel lists](https://developers.google.com/tech-writing/one/lists-and-tables#keep_list_items_parallel) may be useful)?